FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Common/Configurations/StorageConfiguration.cs

```diff
@@ -0,0 +1,18 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Newtonsoft.Json;
+
+namespace Microsoft.Health.Fhir.Synapse.Common.Configurations
+{
+    public class StorageConfiguration
+    {
+        [JsonProperty("externalStorageConnectionString")]
+        public string ExternalStorageConnectionString { get; set; } = string.Empty;
+
+        [JsonProperty("internalStorageConnectionString")]
+        public string InternalStorageConnectionString { get; set; } = string.Empty;
```

Comment:
A question, in what cases we use internal or external connection string?

Reply:
Seems they can be distinguished by **TokenCredentialTypes** config, can we only add a **StorageConnectionString** here?

Reply:
Internal Resources : internal storage for table and queue.
External Resources :  ACR, data lake, FHIR server.

Internal storage could use internal connection string and External data lake could use external connection string. These two strings may be different.